torch_spex/forces.py

```diff
@@ -1,17 +1,21 @@
 import torch
-from typing import List
+from typing import List, Optional
 
 def compute_forces(
-    energy: torch.Tensor, positions: List[torch.Tensor], is_training=True
-) -> List[torch.Tensor]:
+    energy: torch.Tensor, positions: torch.Tensor, is_training: bool = True
+) -> torch.Tensor:
+    grad_outputs : Optional[List[Optional[torch.Tensor]]] = [torch.ones_like(energy)]
     gradient = torch.autograd.grad(
-        outputs=energy,
-        inputs=positions,
-        grad_outputs=torch.ones_like(energy),
+        outputs=[energy],
+        inputs=[positions],
+        grad_outputs=grad_outputs,
         retain_graph=is_training,
         create_graph=is_training,
-    )
-    return [-single_structure_gradient for single_structure_gradient in gradient]
+    )[0]
+    if gradient is None:
+        raise ValueError("something went wrong")
```

Comment:
Might be more helpful for debugging
```suggestion
raise ValueError(
  "Unexpected None value for computed gradient. torch.autograd.grad input arguments:\n"
  f"  outputs={[energy]!r},\n"
  f"  inputs={[positions]!r},\n"
  f"  retain_graph={is_training},\n"
  f"  create_graph={is_training}")
```
This how the error message look like
![image](https://github.com/lab-cosmo/torch_spex/assets/2772557/98f42e23-d259-4e40-87cb-1ef7782a888f)